README.md

```diff
@@ -23,18 +23,13 @@ When upgrading to a new Go version:
 
 ## Changelog
 
- - 2022-10-25 -- Rebuild to update base image for security vulns
- - 2022-09-07 -- Rebuild to update base image for security vuln
+ - 2022-10-27 -- Update to golang 1.19, alpine 3.16. Remove explicit update to libssl and libcrypto
  - 2022-07-18 -- Stop building image for golang 1.16
- - 2022-07-04 -- Rebuild to update base image for security vulns
- - 2022-06-17 -- Rebuild to update base image for security vulns
- - 2022-04-19 -- Rebuild to update base image for security vulns
- - 2022-04-06 -- Rebuild to update base image for security vulns
- - 2022-03-30 -- Rebuild to update base image for security vulns
- - 2022-03-28 -- Rebuild to update base image for security vulns
  - 2022-03-23 -- Explicitly update libssl and libcrypto for security vulns
- - 2022-02-23 -- Rebuild to update base image for security vulns
- - 2022-02-07 -- Rebuild to update base image for security vulns
- - 2022-01-17 -- Rebuild to update base image for security vulns
- - 2021-12-13 -- Rebuild to update base image for security vulns
- - 2021-11-06 -- Rebuild to update base image for security vulns
+
+## Rebuild to update base image for security vulnerabilities
+ - 2022
+   - 25 Oct, 07 Sep, 04 Jul, 17 Jun, 19 Apr, 06 Apr, 30 March, 28 March, 23 Feb, 07 Feb, 17 Jan
+ - 2021
+   - 13 Dec, 06 Nov
+ 
```

Comment:
I'm kinda fine with how this was - makes it easy to glance at the blame if you want to see for what reason specifically we did the rebuild.

Reply:
mind putting your thoughts here? https://countingup.slack.com/archives/C6V7P8P0C/p1666860085465439

Reply:
problem with old way of doing this is that it was getting extremely noisy (now that we do rebuild more often)

Reply:
I don't feel strongly about this - happy to go with what you've changed it to then